scripts/deploy.sh

```diff
@@ -192,15 +192,24 @@ EOF
     exit 1
   fi
 
-  src_path=$(realpath "$simulator_path/src/aoai-simulated-api")
+  image_name="aoai-simulated-api:latest"
+  set +e
+  existing_image=$(az acr repository show --name $acr_name --image $image_name 2>&1)
```

Comment:
It looks like this is expected to be JSON output, in which case that should be specified in the command :-)

```suggestion
  existing_image=$(az acr repository show --name $acr_name --image $image_name --output json 2>&1)
```